Makefile

```
@@ -1,23 +1,26 @@
-PYTHON_VERSIONS := 3.9 3.10 3.11 3.12
+PYTHON_VERSIONS := 3.9 3.10 3.11 3.12 3.13
 
 release:
 	src/release/scripts/release.sh
 
 prep-release:
-	src/release/scripts/prep-release.sh $(PYTHON_VERSIONS)
+	src/release/scripts/prep-release.sh
 
-release/install-dependencies:
-# Install pyenv
-	brew install pyenv
+build-wheels:
+	src/release/scripts/build-wheels.sh $(PYTHON_VERSIONS)
 
-# Install build
-	pip install build --break-system-packages
+release/install-dependencies:
+# Check if pyenv is installed
+	$(MAKE) check-pyenv-is-present
 	
 # Install all the python versions we support in one line
 	pyenv install --skip-existing $(PYTHON_VERSIONS)
 
 # Set pyenv local and install dependencies for each version
 	for version in $(PYTHON_VERSIONS); do \
 		pyenv local $$version; \
-		pyenv exec pip install wheel setuptools; \
-	done
\ No newline at end of file
+		pyenv exec pip install wheel setuptools build; \
```

```suggestion
		pyenv exec pip3 install wheel setuptools build; \
```

For better support across sytems